replication/README.md

```diff
@@ -366,6 +367,18 @@ message GetVolumeReplicationInfoResponse {
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // Holds the last sync duration.
+  // This duration is in seconds.
```

Comment:
This is not correct anymore, `Duration` is more precise as documented in `duration.proto`:
![image](https://github.com/csi-addons/spec/assets/400257/0b3bfad4-a504-4479-bffe-beb673e685ce)

Reply:
Do we need to mention this here? ain't this sufficient?